release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -3574,3 +3574,30 @@ With this release, Cloud Provider {rh-openstack-first} is updated to 1.25.5. The
 ==== Updating
 
 To update an existing {product-title} 4.12 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI] for instructions.
+
+[id="ocp-4-12-15"]
+=== RHBA-2023:2037 - {product-title} 4.12.15 bug fix update
+
+Issued: 2023-05-02
+
+{product-title} release 4.12.15 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2023:2037[RHBA-2023:2037] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2023:2036[RHBA-2023:2036] advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.12.15 --pullspecs
+----
+
+[id="ocp-4-12-15-features"]
+==== Features
+
+[id="ocp-4-12-15-cno-ignored-kuryr-mtu"]
+===== CNO recognizes Kuryr's MTU settings
+
+* Previously, the Cluster Network Operator (CNO) configuration ignored Kuryr's maximum transmission unit (MTU) settings when using the {rh-openstack-first} Networking service (neutron) component to create a network for OpenShift services. CNO would create a network in Neutron with the wrong MTU property, and this action could cause incompatibility issues among network components. With this update, the CNO does not ignore the Kuryr MTU setting when creating the network for services. You can then use the network to host OpenShift services.(link:https://issues.redhat.com/browse/OCPBUGS-4896[*OCPBUGS-4896*])
```

Comment:
Also, I am not 100% on this "when using the {rh-openstack-first} Networking service (Neutron) component" specifically the "(Neutron)" part. I didn't see Neutron worded that way in the docs anywhere. Is there a way to rephrase so it isn't in parenthesis?

Reply:
I found the term in the latest OpenStack docs: https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/17.0/html/networking_guide/networking-overview_rhosp-network  .

I would usually associate backticks with such lowercase component names, but I do not see this stylistic practice used in the OpenStack docs :(. Any idea of writers on OCP Stack? I could confirm with them.

Reply:
Hi All,
OpenStack writer here. :) Darragh asked me about this. 
we include the upstream code name in parenthesis after first mention, for the benefit of people who have those upstream code names burned into the brain. Within those parents, it's lower case, no formatting.
[9:46](https://redhat-internal.slack.com/archives/D056JRNPQG0/p1683035212306119)
OpenStack services (noun)
Description: For definitions of all upstream OpenStack services, see the latest product guide. Here are the [OpenStack services (components) provided by the 16.2 release](https://access.redhat.com/documentation/en-us/red_hat_openstack_platform/16.2/html-single/product_guide/index#sect-components).
The following RHOSP documentation conventions apply:
Use the component name instead of the code name. For example: "Block Storage" instead of "cinder". Exception: "heat templates" as opposed to "Orchestration service templates".
Write the component name in title case.
Write "service" in lowercase if the component name includes "service".
Write the code name in lowercase in parentheses after the component name, and after "service", on the first use of the component name in a module.
Include the word "OpenStack" before the component name only if necessary to differentiate from other product services.
Examples of correct usage: Networking service (neutron), Shared File Systems service (manila), Identity service, OpenStack Networking service.
Use it: yes
Incorrect forms: Block storage service, compute, Horizon

Reply:
Thank you for the help @jamsmithredhat!